parallel-install/pkg/deployment/deletion.go

```diff
@@ -266,54 +266,54 @@ func (i *Deletion) deleteKymaNamespaces(namespaces []string) error {
 
 		go func(ns string) {
 			defer wg.Done()
-			if ns == "kyma-system" {
-				// All the hacks below should be deleted after this issue is done: https://github.com/kyma-project/kyma/issues/11298
-
-				//HACK: Delete finalizers of leftover Secret
-				secret, err := i.kubeClient.CoreV1().Secrets(ns).Get(context.Background(), "serverless-registry-config-default", metav1.GetOptions{})
-				if err != nil && !apierr.IsNotFound(err) {
-					errorCh <- err
-				}
-				if secret != nil {
-					secret.Finalizers = []string{}
-					if _, err := i.kubeClient.CoreV1().Secrets(ns).Update(context.Background(), secret, metav1.UpdateOptions{}); err != nil {
+			// All the hacks below should be deleted after this issue is done: https://github.com/kyma-project/kyma/issues/11298
+			//HACK: Delete finalizers of leftover Secret
+			secrets, err := i.kubeClient.CoreV1().Secrets(ns).List(context.Background(), metav1.ListOptions{LabelSelector: "serverless.kyma-project.io/config=credentials"})
+			if err != nil && !apierr.IsNotFound(err) {
+				errorCh <- err
+			}
+			if secrets != nil {
+				for _, secret := range secrets.Items {
+					secret.SetFinalizers(nil)
+					if _, err := i.kubeClient.CoreV1().Secrets(ns).Update(context.Background(), &secret, metav1.UpdateOptions{}); err != nil {
 						errorCh <- err
 					}
 					i.cfg.Log.Infof("Deleted finalizer from Secret: %s", secret.Name)
 				}
+			}
 
-				//HACK: Delete finalizers of leftover Custom Resources
-				crds, err := i.apixClient.CustomResourceDefinitions().List(context.Background(), metav1.ListOptions{})
-				if err != nil {
-					errorCh <- err
-				}
+			//HACK: Delete finalizers of leftover Custom Resources
+			crds, err := i.apixClient.CustomResourceDefinitions().List(context.Background(), metav1.ListOptions{})
```

Comment:
i think that only crds installed by kyma should be listed here. we label our CRDs